book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
```

Comment:
- [ ] Please make sure that the add rental takes person and book as parameters and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` as you see below :point_down: 
  
    ```ruby
    def add_rental(person, date)
        Rental.new(date, self, person)
    end
    ```